Bottom: 98d26ef0481ea781934ecc7cb72f3b4c074a2a00
Top:    b2f8f59592d0a2d80736daecdcce21a381716e8f
Author: Jason Wang <jasowang@redhat.com>
Date:   2018-07-11 17:38:33 +0800

Refresh of net-skbuff-introduce

---

diff --git a/drivers/net/tun.c b/drivers/net/tun.c
index 6d9f3b3..8568cd7 100644
--- a/drivers/net/tun.c
+++ b/drivers/net/tun.c
@@ -1076,7 +1076,7 @@ static struct sk_buff *tun_prepare_xdp_skb(struct sk_buff *skb)
 	/* TODO: if the program won't modify the packet, avoid the
 	 * skb_copy() here */
 	if (skb_shared(skb) || skb_cloned(skb)) {
-		nskb = skb_copy_headerlen(skb, GFP_ATOMIC, XDP_PACKET_HEADROOM);
+		nskb = skb_copy(skb, GFP_ATOMIC);
 		return nskb;
 	}
